src/commands/authentication.cpp

```diff
@@ -232,3 +241,36 @@ void AOClient::cmdLogout(int argc, QStringList argv)
     moderator_name = "";
     sendPacket("AUTH", {"-1"}); // Client: "You were logged out."
 }
+
+void AOClient::cmdChangePassword(int argc, QStringList argv)
+{
+    QString username;
+    QString password;
+    if (argc == 1) {
+        if (moderator_name.isEmpty())
+            return;
```

Comment:
might be a good idea to tell the user they can't change their password because they aren't logged in rather than just eating the command and not telling them why